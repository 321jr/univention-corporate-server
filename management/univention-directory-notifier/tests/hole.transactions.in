1 dc=example,dc=com a
3 dc=example,dc=com m
