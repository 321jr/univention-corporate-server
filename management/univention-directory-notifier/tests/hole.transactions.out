1 dc=example,dc=com a
2 dc=example,dc=com m
3 dc=example,dc=com m
